pkg/crc/preflight/preflight_windows.go

```diff
@@ -55,11 +55,61 @@ var hypervPreflightChecks = [...]PreflightCheck{
 	},
 }
 
+var traySetupChecks = [...]PreflightCheck{
+	{
+		checkDescription: "Checking if daemon service is installed",
+		check:            checkIfDaemonServiceInstalled,
+		fixDescription:   "Installing daemon service",
+		fix:              fixDaemonServiceInstalled,
+		flags:            SetupOnly,
+	},
+	{
+		checkDescription: "Checking if daemon service is running",
+		check:            checkIfDaemonServiceRunning,
+		fixDescription:   "Starting daemon service",
+		fix:              fixDaemonServiceRunning,
+		flags:            SetupOnly,
+	},
+	{
+		checkDescription: "Checking if tray binary is present",
+		check:            checkTrayBinaryExists,
+		fixDescription:   "Caching tray binary",
+		fix:              fixTrayBinaryExists,
+		flags:            SetupOnly,
+	},
+	{
+		checkDescription: "Checking if tray binary is added to startup applications",
+		check:            checkTrayBinaryAddedToStartupFolder,
+		fixDescription:   "Adding tray binary to startup applications",
+		fix:              fixTrayBinaryAddedToStartupFolder,
+		flags:            SetupOnly,
+	},
+	{
+		checkDescription: "Checking if tray version is correct",
+		check:            checkTrayBinaryVersion,
+		fixDescription:   "Caching correct tray version",
+		fix:              fixTrayBinaryVersion,
+		flags:            SetupOnly,
+	},
+	{
+		checkDescription: "Checking if tray is running",
+		check:            checkTrayRunning,
+		fixDescription:   "Starting CodeReady Containers tray",
+		fix:              fixTrayRunning,
+		flags:            SetupOnly,
+	},
+}
```

Comment:
@anjannath don't we also need the cleanup steps so `crc cleanup` can also deal with removing the tray part on windows?

Reply:
Yes, that is needed, i was thinking to create another PR for it, but i can push those commits here also, haven't finished the cleanup part yet.

Reply:
Sure, better to have it part of this one but a separate commit would be OK.

Reply:
@praveenkumar Added the cleanup part in https://github.com/code-ready/crc/pull/1221/commits/b2fa823e398c20c6ea60b3b1645de282e7f229c5